src/app/app.component.ts

```diff
@@ -1,10 +1,30 @@
 import { Component } from '@angular/core';
+import { Product } from './models/product';
+import { ProductService } from './service/product.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
 export class AppComponent {
-  title = 'app';
+  products: Product[];
+  title = 'Products';
+
+  constructor(private productService: ProductService) {
+    this.getProducts();
+  }
+
+  getProducts(): void {
+    this.productService.getAll()
+      .subscribe(products => {
+        console.log(products)
+        this.products = products;
+      });
+  }
+  showProduct(id): void {
```

Comment:
Why do we have this if it's supposed to be taken care of in a different story? 🤔  -- #FeatureCreepMaybe